notebooks/official/workbench/pricing_optimization/pricing-optimization.ipynb

```diff
@@ -0,0 +1,2206 @@
+{
```

Comment:
<p>move title to link cell.</p><p>make title more descriptive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>